prime-dotnet-webapi/Controllers/EnrolmentController.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Models;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/v1/[controller]")]
```

Comment:
This is fine for now, but we should have a bigger conversation about api and version in the pathing

Reply:
After discussion the API versioning will be handled by the routing in OpenShift, so it does not need to be in the code.  From a dotnet routing perspective, there will be no version in the endpoint URL.